recipes/eups/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "eups" %}
+{% set version = "2.1.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RobertLuptonTheGood/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 095e18b6079c49bdba48355edcb479b0b1b91a9013cafdf00ca112e596b0885b
+
+build:
+  skip: True  # [win or py < 37]
+  number: 0
+
+requirements:
+  build:
+    - make
+
+  host:
+    - python
+    - git
+    - make
+
+  run:
+    - python
+
+about:
+  home: https://github.com/RobertLuptonTheGood/eups
+  license: GPL-2.0-or-later
+  license_family: GPL
+  license_file: gpl.txt
+  summary: 'A version manager tracking product dependencies'
+
+extra:
+  recipe-maintainers:
+    - gcomoretto
+    - brianv0
```

Comment:
It should be possible to include the eups tests by adding:
```
test:
  source_files:
    tests
```
and then adding `python tests/testAll.py` to `run_test.sh`